node_io/lib/src/streams.dart

```diff
@@ -204,4 +211,13 @@ class NodeIOSink extends WritableStream<List<int>> implements IOSink {
   void writeln([Object obj = ""]) {
     _write(encoding.encode("$obj\n"));
   }
+
+  @override
+  add(List<int> data) {
+    // Add as buffer
+    if (!(data is Uint8List)) {
```

Comment:
Can use `data is! Uint8List` here as well.

Reply:
ok